### PR TITLE
DNS during enable script

### PR DESCRIPTION
**Pull request by ogra1**
- make sure we have working DNS resolution when the enable.sh script runs, it calls "apt update" now and needs working network access
- also drop "sudo apt update" from the README.md this happens automatically now

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Run the following commands on an all-snap image:
 
     $ sudo snap install --beta --devmode classic
     $ sudo classic
-    (classic)ubuntu@localhost:~$ sudo apt update
```
Comment (mvo5) [inline README.md]: Why dropping it? Its there to show that apt is available in the classic environment.

Comment (ogra1) [inline README.md]: ah, well, let me re-add it then, i meant to show that you dont need to run it anymore ...

Comment (ogra1) [inline README.md]: BTW. is that "Future" block at the bottom of README.md still relevant ?

Comment (mvo5) [inline README.md]: Maybe `apt install hello` then or something? Just some example to show that its a "regular" system :)

**Pull request merged**